### PR TITLE
New endpoints for Business Address

### PR DESCRIPTION
**Pull request by eunicegalang**
Updated endpoints for current users of Business Accounts, and added new endpoints for updating addresses by userId.

### CODE REVIEW

File: src/main/kotlin/com/paysera/lib/mokejimai/clients/MokejimaiApiClient.kt [hunk 1]
```diff
@@ -42,15 +42,23 @@ class MokejimaiApiClient(
...
-        return apiClient.updateUserAddress(userAddress)
+    fun updateCurrentUserAddress(userAddress: UserAddress): Deferred<UserAddress> {
+        return apiClient.updateCurrentUserAddress(userAddress)
     }
 
     fun getUserAccounts(userId: Int): Deferred<MetadataAwareResponse<UserAccount>> {
         return apiClient.getUserAccounts(userId)
     }
+
+    fun getUserAddresses(userId: Int): Deferred<MetadataAwareResponse<UserAddress>> {
```
Comment (vytautasgimbutas) [inline src/main/kotlin/com/paysera/lib/mokejimai/clients/MokejimaiApiClient.kt:57]: userId must be String

Comment (edgar-zigis) [reply src/main/kotlin/com/paysera/lib/mokejimai/clients/MokejimaiApiClient.kt]: it does not really matter, because it goes into path, not as body param

Comment (edgar-zigis) [reply src/main/kotlin/com/paysera/lib/mokejimai/clients/MokejimaiApiClient.kt]: ahh, it was identifier

Comment (vytautasgimbutas) [reply src/main/kotlin/com/paysera/lib/mokejimai/clients/MokejimaiApiClient.kt]: Yep, it's not numeric.

**Pull request merged**